source/includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs

```diff
@@ -130,6 +130,17 @@ public static List<Guitar> GeoWithinSearch()
         return result;
     }
 
+    public static List<Guitar> InSearch()
+    {
+        // start-in-search
+        var guitarList = new[] { "Fender", "Gibson" };
+        var result = guitarsCollection.Aggregate()
+            .Search(Builders<Guitar>.Search.In(g => g.Make, guitarList))
```

Comment:
minor: could use new collection expression syntax:
`.Search(Builders<Guitar>.Search.In(g => g.Make, ["Fender", "Gibson"]))`